### PR TITLE
Inline widget updates & fixes

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: static/inline/js/inline.js [hunk 1]
```diff
@@ -219,7 +220,8 @@
                         breakpoint: 769,
                         settings: {
                             arrows: false,
-                            centerPadding: '0px',
+                            centerPadding: Utils.isset(generalOptions.product_holder_slide_center_padding) ? generalOptions.product_holder_slide_center_padding :'25% 0 0',
```
Comment (rfornes) [inline static/inline/js/inline.js]: Shouldn't the option `product_holder_slide_center_padding` default value actually be "25% 0 0"?
This way you just pass the option and that's it.

`centerPadding: generalOptions.product_holder_slide_center_padding`

Comment (karloscastaneda) [reply static/inline/js/inline.js]: 👍  will update

File: static/inline/js/inline.js [hunk 2]
```diff
@@ -360,7 +363,8 @@
                             breakpoint: 769,
                             settings: {
                                 arrows: false,
-                                centerPadding: '0px',
+                                centerPadding: Utils.isset(options, 'videos_carousel_center_padding') ? options.videos_carousel_center_padding : '35% 0 0',
```
Comment (rfornes) [inline static/inline/js/inline.js]: This should read as:

`centerPadding: options.videos_carousel_center_padding`

`videos_carousel_center_padding` should exists in `defaults/` and it should have a default value

Comment (karloscastaneda) [reply static/inline/js/inline.js]: updating

**Pull request closed without merging**